test/test_browser.py

```diff
@@ -4728,6 +4728,7 @@ def test_pthread_hello_thread(self, opts, modularize):
   # Tests that a pthreads build of -sMINIMAL_RUNTIME works well in different build modes
   @parameterized({
     '': ([],),
+    'growable_arraybuffers': (['-sGROWABLE_ARRAYBUFFERS'],),
```

Comment:
This doesn't enable memory growth - is this still testing what we want?

Reply:
Oops, wrong test.  Fixed.